data/country/metadata/qt-schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
```

Comment:
Why is the use of qt-schema.json file

Reply:
The current json schema is not supported by quicktype. Hence modified this manually to create a new schema which can used with QT